Makefile

```diff
@@ -282,7 +282,7 @@ endif
 #
 
 # Define a list of headers to install. The default is to only install blis.h.
-HEADERS_TO_INSTALL := $(BLIS_H_FLAT)
+HEADERS_TO_INSTALL      := $(BLIS_H_FLAT)
 
 # If CBLAS is enabled, we also install cblas.h so the user does not need to
 # change their source code to #include "blis.h" in order to access the CBLAS
```

Comment:
```suggestion
# change their source code to #include <blis.h> in order to access the CBLAS
```
But this comment seems out of place.

Reply:
Yes, this was likely a copy-paste bug. Thanks Jed, I'll fix it.

Reply:
I just looked into this, and the mention of `blis.h` is actually *not* a typo. What it's trying to say is the following:

If the user already has an application that has `#include "cblas.h"` and wants to use BLIS as a CBLAS implementation, then we have to install the `cblas.h` header in order to provide the best portability experience. If we didn't install the `cblas.h` header, the user could *still* use BLIS, but they'd have to update their application source code to use `#include "blis.h"` instead of `#include "cblas.h"`.

I'll try to update the comment to clarify.